Automattic/jetpack
# Blogroll: Update blog appender styling and functionality

Fixes https://github.com/Automattic/wp-calypso/issues/82173

## Proposed changes:

* Updates blog appender edge case based on design feedback

Replaces blog appender placeholder site icon
Updates message if no results are found in blog appender

<img width="293" alt="CleanShot 2023-09-26 at 23 00 01@2x" src="https://github.com/Automattic/jetpack/assets/10482592/98753eb4-4f48-4624-a593-e4c57be688b0">

<img width="287" alt="CleanShot 2023-09-26 at 23 00 49@2x" src="https://github.com/Automattic/jetpack/assets/10482592/b2e6ad83-d094-4367-98b7-0ec383db11bc">

Designs: aOWo8Hpzper3EIXwPbuyzU-fi-361_3369


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Checkout branch
* Navigate to the editor and add blogroll block
* Open the blogroll appenderVerify placeholder site icon has been updated to match designs
* Open the blogroll appender, search for a random site, and verify the "No results" message matches the designs
